stdlib/shutil.pyi

```diff
@@ -91,7 +91,12 @@ class _RmtreeType(Protocol):
         ) -> None: ...
         @overload
         def __call__(
-            self, path: StrOrBytesPath, ignore_errors: bool = False, *, onexc: _OnExcCallback, dir_fd: int | None = None
+            self,
+            path: StrOrBytesPath,
+            ignore_errors: bool = False,
+            *,
+            onexc: _OnExcCallback | None = None,
+            dir_fd: int | None = None,
         ) -> None: ...
```

Comment:
I'm not sure why we need this *and* the first overload. The first overload seems redundant.

Reply:
Agreed, given the params are optional. Good catch!